packages/flutter_tools/test/integration.shard/command_output_test.dart

```diff
@@ -55,6 +56,20 @@ void main() {
     expect(result.stdout, contains('Running shutdown hooks'));
   });
 
+  test('flutter config contains all features', () async {
```

Comment:
Will this test pass on release branches?

Reply:
Yes - because we need to create arguments before we know what the current flutter version is, all of the features are always displayed